package.json

```diff
@@ -67,6 +68,7 @@
     }
   ],
   "devDependencies": {
+    "@changesets/cli": "^2.27.10",
```

Comment:
Huh I thought this was maybe baked onto pnpm. Is it a peerdep?

Reply:
I just followed the pnpm change set guide, it advised this